src/utils/execution.ts

```diff
@@ -286,3 +286,9 @@ export const buildSafeTransaction = (template: {
         nonce: template.nonce,
     };
 };
+
+export const buildRawError = (errorMessage: string) => {
+    const errorSelector = "0x08c379a0"; // ERC 838
```

Comment:
looks like 838 was an EIP, search term `erc 838` doesn't find anything useful in google

Reply:
Why is the selector hardcoded inside of the util function? This limits the scope of usage only to this error.

It either should be passed as an argument, or the function should be placed inside the test file where it's used

Reply:
> looks like 838 was an EIP, search term `erc 838` doesn't find anything useful in google

It should be an ERC, I'm working on that but I'll change in the meantime: https://github.com/ethereum/EIPs/pull/7186

Reply:
> Why is the selector hardcoded inside of the util function? This limits the scope of usage only to this error.
> 
> It either should be passed as an argument, or the function should be placed inside the test file where it's used

It is the selector for 'Error(string)' which is the only one used in solidity up to 0.8,

Do you want it to be more generic so it is ready for the eventual adoption of soldity ^0.8?

Reply:
We want our contracts to be compilable with both 0.7.x and 0.8.x

Reply:
You could also rename the function to clarify that it's only used for Error strings. Page 108 from https://docs.soliditylang.org/_/downloads/en/latest/pdf/

Reply:
Just sent a new verson, the mock contract bubbles the message so it is possible to use the chai-matchers revertWith and revertWithCUstomError